### PR TITLE
Milestone 2 - Projects & About Sections

### PR DESCRIPTION
**Pull request by chelmerrox**
Hi! This PR is for milestone 2. I've chosen template 5 from Figma as my design guide. I have completed its requirements to:

1. Add the Projects section (Section 2)
2. Add the About section (Section 3)
3. Use `<ul>` tags for: a) Project section   b) About section
4. Add the :active, :hover pseudoclasses to the buttons in all sections
5. Stick to the design etc of the template
6. Use flexbox and grid where necessary (in both sections)

Thanks in advance for your review, comments & help!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,6 +43,191 @@ <h1 class="section-1-heading-mobile playfair-display-font font-color">Losalini R
...
+        </div>
+      </section>
+
+     <!--Section 3 | About Section for mobile screens-->
+     <section class="section-3-mobile">
+      <h2 class="section-3-heading-mobile playfair-display-font font-color">About</h2>
+      <p class="section-3-paragraph-mobile poppins-font font-color">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, 
+        quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+      
+      <div class="section-3-button-mobile">
```
Comment (muneebulrehman) [inline index.html:181]: If you refer to Figma template 5, the about section has padding on the left and right sides, but I think you forgot to add it. I recommend adding some padding. Preferably follow the template provided. 😊

File: index.html [hunk 2]
```diff
@@ -41,6 +43,191 @@ <h1 class="section-1-heading-mobile playfair-display-font font-color">Losalini R
...
+              <h3 class="poppins-font font-white font-weight-lighter">Project name goes here</h3>
+
+              <ul class="project-technologies font-white">
+                <li class="font-color mr-3">HTML/CSS</li>
+                <li class="font-color mr-3">Ruby on Rails</li>
+                <li class="font-color">JavaScript</li>
+              </ul>
+
+              <div class="see-this-project-btn-mobile">
+                <a href="#" class="poppins-font bg-color font-white">See this project <span class="material-icons ">arrow_forward</span></a>
```
Comment (muneebulrehman) [inline index.html:68]: Hey @chelmerrox, when hovering over the button, it shows different background for text and the rest of the button. Kindly fix this bug. Also, the button should have some interaction when it is pressed. I would recommend adding it. The same goes for the rest of the buttons in the Project section.
errorhover

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @chelmerrox , 

Amazingly done, I love your CSS writing style.
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Great CSS code
- Used Semantic HTML tags
- Followed Figma template chosen
- Professional Readme

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Your commit messages are too long. You can consider stortening them.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
**STATUS** **APPROVED** 🎉 🎉 

Hi @chelmerrox  ,

Good job on making the requested changes by previous code reviewer your project is complete. There is nothing else to say than it's time to merge it. Congratulations 🎉 

### Highlights
- [x] Good and Descriptive Pull request.
- [x] No linter errors.
- [x] Nice User interface.
- [x] Neat file structure.
- [x] Nice styling keep it up.

### Required Changes
N/A

Check the comments under the review.

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!!! 🥇 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**